frontend/src/App.tsx

```diff
@@ -12,12 +12,30 @@ import ProtectedRoute from 'routes/protected.route'
 // Contexts
 import AuthContextProvider from 'contexts/auth.context'
 
+// Service
+import { initializeGA, sendPageView } from 'services/ga.service'
+
 import './styles/app.scss'
 
+const GA = () => {
```

Comment:
Could you explain why this is initialized here instead of within the authcontextprovider? I’m not too sure where it should be initialised so an explanation would be very helpful

Reply:
This example initialized it in the app actually https://github.com/react-ga/react-ga/wiki/Using-history-for-GA-Tracking#implementation-1---logpageview-appjs